ospatch/apt_upgrade.go

```diff
@@ -89,12 +91,12 @@ func RunAptGetUpgrade(opts ...AptGetUpgradeOption) error {
 		pkgNames = append(pkgNames, pkg.Name)
 	}
 	logger.Infof("Updating %d packages.", len(pkgNames))
-	logger.Debugf("Packages to be installed: %s", fPkgs)
+	logger.Infof("Packages to be installed: %s", fPkgs)
```

Comment:
It may also be clearer to combine this with the above log line since they're both `Info` now. Maybe` "Updating %d packages: %s", len(pkgNames), fPkgs `. Or you could go further and combine it with the "dryrun" message too. So for dryruns, log `Dryrun, not updating %d packages: %s", len(pkgNames), fPkgs `.

And fyi that people may code against this. So even if we add structured logging in the future, we may need to keep this log message around.

Reply:
Ack on the structured logging. What I think i may end up doing is keeping the message field as the human readable form, and then adding more structure around that. So still one log entry just with both this message and also more structured fields.